_install-and-configure/install-opensearch/index.md

```diff
@@ -60,8 +60,7 @@ Port number | OpenSearch component
 443 | OpenSearch Dashboards in AWS OpenSearch Service with encryption in transit (TLS)
 5601 | OpenSearch Dashboards
 9200 | OpenSearch REST API
-9250 | Cross-cluster search
-9300 | Node communication and transport
+9300 | Node communication and transport (Cluster internal), Cross Cluster Search
```

Comment:
What do we mean by "Cluster internal"? Is this port for the internal transport layer only?

```suggestion
9300 | Node communication and transport (internal cluster), Cross Cluster Search
```

Reply:
Should cross cluster search be lower case?

Reply:
> What do we mean by "Cluster internal"? Is this port for the internal transport layer only?

Yes

Reply:
Change suggested by @Naarcha-AWS  looks good to me